admin/registrations.py

```diff
@@ -0,0 +1,108 @@
+from flask import (session, render_template, url_for, flash, redirect)
+import boto.ses
+from admin import app
+from performanceplatform.client.admin import AdminAPI
+from admin.forms import AboutYouForm, AboutYourServiceForm
+from admin.helpers import base_template_context
+
+REGISTER_ROUTE = '/register'
```

Comment:
We had a look at the other routes in the admin app and they don't handle a trailing slash, so we decided to remain consistent and not to handle it.